core/jvm/src/main/scala/cats/effect/Repl.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+
+object Repl {
+  private lazy val yoloRuntime: IORuntime = IORuntime.default
+  implicit class Yolo[A](private val self: IO[A]) extends AnyVal {
+    def yolo: A = yoloRuntime.unsafeRunSync(self)
+  }
```

Comment:
LOL. Maybe just `unsafeRunSync()`? :-)

Reply:
Shrug, I liked `def !(): A` but Doobie, Http4s, and @systemfw all use `.yolo` for this. No strong opinions really.

Reply:
@rossabaker pointed out that folks have copy/pasted YOLO examples from http4s docs in to real code, so I named the object `Repl` to make it clear it was not intended for production use. I worry a little about overloading the meaning of `unsafeRunSync` as folks will see that in REPL and mdoc examples and get confused when that method doesn't exist on `IO`. Hence, I think we should name it something other than `unsafeRunSync`.